### ISSUE TITLE
[Track2: ReID] Question about 9 pairs of features, combination of losses, and some missing codes.

### ISSUE THREAD

**Issue by bismex** [he010103/Traffic-Brain#12]
Dear Zhiqun He,

I am getting a lot of help for you providing the code. I have some questions about that code. 

1) Could you tell me how to extract 9 pairs of features for query and gallery in the provided google drive?

Through the codes of run.sh and submit.py, I can get a pair of features for query and test (gallery). However, in the gen_track2.py code, the features of the paired query and test (gallery) I learned are not loaded. To be more specific, 9 pairs of pre-computed (downloadable) features for queries and tests (gallery) are loaded, and the final result is output by these feature vectors. In other words, the gen_track2.py code provides for testing pre-trained features (9 pairs) rather than using features I trained. I know that I can download these feature vectors in your google drive, but I don't think there is a way to extract these 9 pairs of features in the code and workshop paper. I wonder how to extract 9 pairs of features. Are they about changing losses? or Are they about changing training datasets? Could you tell me how to extract them?

2) There are several options for the combination of loss in the code. Could you tell which loss combination has the best performance?

When you submit to the challenge, I want to know if you use the loss combination below.
(if cfg.MODEL.IF_WITH_CENTER == 'yes' -> centroid mode)
cfg.MODEL.METRIC_LOSS_TYPE =? 
- 'center' -> CE + center
- 'range_center' -> CE + center + range
- 'triplet_center' -> CE + triplet + track_triplet + center
- 'triplet_range_center' -> CE + triplet + track_triplet + center + range

3) The code for "Reranking with spatial-temporal cue" seems to be missing. Any plans to release this code?

Looking at the gen_track2.py code, there is the following code.
"from metrics import track_reranking_weight_feat"
However, I haven't found any metrics folder or track_reranking_weight_feat function anywhere. Isn't that function provided?

Thanks
Seokeon Choi
